src/common/registry/storage/key.storage.ts

```diff
@@ -16,6 +17,19 @@ export class RegistryKeyStorageService {
     return await this.repository.find(where, options);
   }
 
+  findStream(where: FilterQuery<RegistryKey>, fields?: string[]): AsyncIterable<RegistryKey> {
+    const knex = this.repository.getKnex();
+    const stream = knex
+      .select(fields || '*')
+      .from<RegistryKey>('registry_key')
+      .where(where)
```

Comment:
I think we need orderBy here to be sure the results are not shuffled

Reply:
I believe it's not mandatory because it's a stream without requesting batch of data.

Reply:
added